.github/draft-release-notes-config.yml

```diff
@@ -12,11 +12,17 @@ replacers:
   - search: '##'
     replace: '###'
 
+exclude-labels:
+  - 'skip-changelog'
+
+exclude-contributors:
```

Comment:
do we want to exclude trigger bot? What about backported changes? Especially now that everyone is working off main branch